main.go

```diff
@@ -217,3 +242,28 @@ func filterVariants(module, variant string, variantsMap gradle.Variants) (gradle
 	}
 	return filteredVariants, nil
 }
+
+func parseQuarantinedTests(input string) ([]string, error) {
+	if input == "" {
+		return nil, nil
+	}
+
+	quarantinedTests, err := testquarantine.ParseQuarantinedTests(input)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse quarantined tests input: %w", err)
+	}
+
+	var skippedTests []string
+	for _, qt := range quarantinedTests {
+		if len(qt.TestSuiteName) == 0 || qt.TestSuiteName[0] == "" || qt.ClassName == "" || qt.TestCaseName == "" {
+			continue
+		}
+
+		packageName := qt.TestSuiteName[0]
```

Comment:
TODO: check if qt.TestSuiteName stores the package name or it is part of qt.ClassName